webapp/src/projects.tsx

```diff
@@ -821,12 +821,15 @@ export class ProjectsDetail extends data.Component<ProjectsDetailProps, Projects
             case "forumUrl":
                 icon = "comments"
                 break;
+            case "forumExample":
+                icon = "pencil"
+                break;
             case "template":
             default:
                 if (youTubeId || youTubePlaylistId) icon = "youtube";
                 break;
         }
-        return this.isLink() && type != "example" // TODO (shakao)  migrate forumurl to otherAction json in md
+        return this.isLink() && type != "forumExample" // TODO (shakao)  migrate forumurl to otherAction json in md
```

Comment:
what about existing "example" cards?

Reply:
no change in those--they still have the blocks/text/js. ultimately we can just merge forumExample and sharedExample (https://github.com/microsoft/pxt/pull/7352) but it's messy to maintain compatibility between beta/live so i'm leaving it for now